.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 20.17.0
```

Comment:
I couldn't get `asdf` to read the legacy `.node-version` file, even after adding `legacy_version_file = yes` to my `~/.asdfrc`. The only way I got it to work was using the new `.tool-versions` format. 

@manuelpuchta Would it be OK if we switch from the legacy file to the new one, or should we just keep around both?

Reply:
As discussed, let's switch to `.tool-versions`, also in the pipeline config, please.